scaleway/resource_k8s_cluster_beta.go

```diff
@@ -53,7 +53,7 @@ func resourceScalewayK8SClusterBeta() *schema.Resource {
 			"ingress": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Default:     "no_ingress",
+				Default:     "none",
```

Comment:
Please use constant from sdk package. This will avoid similar issue in the future.

Reply:
There is none yet :sweat_smile: